app/models.py

```diff
@@ -226,6 +216,23 @@ def set_permissions(self):
             self.can_send_international_sms = INTERNATIONAL_SMS_TYPE in [p.permission for p in self.permissions]
 
 
+class ServicePermission(db.Model):
+    __tablename__ = "service_permissions"
+
+    service_id = db.Column(UUID(as_uuid=True), db.ForeignKey('services.id'),
+                           primary_key=True, index=True, nullable=False)
+    permission = db.Column(db.String(255), db.ForeignKey('service_permission_types.name'),
+                           index=True, primary_key=True, nullable=False)
+    service = db.relationship("Service", foreign_keys=[service_id])
+    created_at = db.Column(db.DateTime, default=datetime.datetime.utcnow, nullable=False)
+
+    service_permission_types = db.relationship(
+        Service, backref=backref("permissions", cascade="all, delete-orphan"))
```

Comment:
What does `delete-orphan` do here exactly? Just want to make sure we know the difference to avoid any unexpected behaviour?